/oak:
  ros__parameters:
    camera:
      i_enable_imu: true
      i_enable_ir: true
      i_floodlight_brightness: 0
      i_ip: ''
      i_laser_dot_brightness: 800
      i_mx_id: ''
      i_nn_type: spatial
      i_pipeline_type: Depth
      i_usb_speed: SUPER_PLUS
    left:
      i_board_socket_id: 1
      i_fps: 30.0
      i_height: 720
      i_max_q_size: 30
      i_publish_topic: false
      i_resolution: '720'
      i_width: 1280
      r_exposure: 1000
      r_iso: 800
      r_set_man_exposure: false

    right:
      i_board_socket_id: 2
      i_fps: 30.0
      i_height: 720
      i_max_q_size: 30
      i_publish_topic: false
      i_resolution: '720'
      i_width: 1280
      r_exposure: 1000
      r_iso: 800
      r_set_man_exposure: false
    stereo:
      i_align_depth: true
      i_bilateral_sigma: 0
      i_board_socket_id: 0
      i_decimation_filter_decimation_factor: 1
      i_decimation_filter_decimation_mode: PIXEL_SKIPPING
      i_depth_filter_size: 7
      i_depth_preset: HIGH_ACCURACY
      i_enable_decimation_filter: false
      i_enable_distortion_correction: false
      i_enable_spatial_filter: false
      i_enable_temporal_filter: false
      i_enable_threshold_filter: false
      i_extended_disp: false
      i_height: 720
      i_lr_check: true
      i_lrc_threshold: 10
      i_max_q_size: 30
      i_rectify_edge_fill_color: 0
      i_spatial_filter_alpha: 0.5
      i_spatial_filter_delta: 20
      i_spatial_filter_hole_filling_radius: 2
      i_spatial_filter_iterations: 1
      i_stereo_conf_threshold: 255
      i_temporal_filter_alpha: 0.4000000059604645
      i_temporal_filter_delta: 20
      i_temporal_filter_persistency: VALID_2_IN_LAST_4
      i_threshold_filter_max_range: 15000
      i_threshold_filter_min_range: 400
      i_width: 1280
    use_sim_time: false
